### PR TITLE
Tutorial: Sentiment Analysis on French Tweets using MySQL

### PR DESCRIPTION
**Pull request by harikishantk**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #4873 

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
@harikishantk 
If you completed this PR, please mark it as `Ready for review`.

**Comment by martyna-mindsdb** [comment]
@harikishantk 
Are you still working on it?

**Comment by harikishantk** [comment]
@martyna-mindsdb 
Ready for review!

File: docs/tutorials/sentiment-analysis-on-french-tweets-using-mysql.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```
+SELECT  *
+FROM mindsdb.models
+WHERE  name='sentiment_analyzer';
+```
+After a few seconds you should see the status change from **generating** to **complete.**
+
+## Making predictions
+
+Once the status is complete, the behavior is the same as with any other AI table you can query it or by directly specifying your inputs, like so:
```
Comment (martyna-mindsdb) [inline docs/tutorials/sentiment-analysis-on-french-tweets-using-mysql.mdx:108]: Actually, you could include both single prediction and batch predictions as follows:
 
`...the behavior is the same as with any other AI table. You can query it directly by specifying your inputs:
SELECT  *  FROM mindsdb.sentiment_analyzer
WHERE  text='Je vais me crier après avoir regardé Marley et moi.';

Or by joining with the data table for batch predictions:
SELECT  *  FROM mindsdb.sentiment_analyzer as p
JOIN mysql.french_tweets as d;`

**Comment by martyna-mindsdb** [comment]
@harikishantk 
Please see one review comment above.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@harikishantk 
All is fine now.

Please sign our CLA so we can merge your PR. You can do that by leaving a comment here that contains this line: I have read the CLA Document and I hereby sign the CLA

**Comment by harikishantk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
recheckcla

**Pull request merged**